website/docs/index.html.markdown

```diff
@@ -213,15 +277,17 @@ In addition to [generic `provider` arguments](https://www.terraform.io/docs/conf
 * `max_retries` - (Optional) Maximum number of times an API call is retried when AWS throttles requests or you experience transient failures.
   The delay between the subsequent API calls increases exponentially.
   If omitted, the default value is `25`.
-  Can also be set using the environment variable `AWS_MAX_ATTEMPTS`.
-* `profile` - (Optional) AWS profile name as set in the shared credentials file.
-* `region` - (Optional) AWS region. Can also be set with either the `AWS_REGION` or `AWS_DEFAULT_REGION` environment variables, or via a shared config file if `profile` is used.
+  Can also be set using the environment variable `AWS_MAX_ATTEMPTS`
+  and the shared configuration parameter `max_attempts`.
+* `profile` - (Optional) AWS profile name as set in the shared configuration and credentials files.
+  Can also be set using either the environment variables `AWS_PROFILE` or `AWS_DEFAULT_PROFILE`.
```

Comment:
```suggestion
* `profile` - (Optional) AWS profile name as set in the shared configuration and credentials files. Can also be set using either the environment variables `AWS_PROFILE` or `AWS_DEFAULT_PROFILE`.
```
I think the more common approach is all on one line, which I like because lines can be moved (alphabetized) without leaving straggling bits and pieces.

Reply:
I've been using Semantic Line Breaks (https://sembr.org) for clean diffs and maintainability.